src/Uno.UI/Controls/Window.macOS.cs

```diff
@@ -129,11 +129,9 @@ public override void SendEvent(NSEvent evt)
 					case NSEventType.ScrollWheel:
 						CoreWindowEvents?.RaisePointerWheelChanged(ToPointerArgs(evt, posInWindow));
 						break;
-
-					default:
-						base.SendEvent(evt);

```

Comment:
Oops!

Reply:
Yeah @davidjohnoliver, we don't want a default handler in the `case` instead we want to call the base all the time.

Reply:
There isn't any need to filter the events based on the actual usage in uno ?

Reply:
Yesterday when synced with @dr1rrb we agreed to call base for all the different events (without filtering) and check if this brings any performance hit as we could be executing the pointer event twice. 

I ran the sample app with this chance and did not notice any improvement degradation.

@jeromelaban